rtf

```diff
@@ -298,6 +304,15 @@ class Vagrant(Host):
   def start(self):
     """ Setup and run this host """
     proxy = have_proxy()
+    if (proxy==""):
+      command = "vagrant status " + self.hostid
```

Comment:
i don't understand what this is for

Reply:
When offline we can only try to start a group of machines that are shutdown.
This ensure the requested vms are just in poweroff mode and not in the need to be installed completely.